SQL/Archive/17.0/2016-09-18-Config_add_content_security.sql

```diff
@@ -0,0 +1,4 @@
+-- Insert necessary values into configsettings and config
+
+INSERT INTO ConfigSettings (Name,Description,Visible,AllowMultiple,DataType,Parent,Label,OrderNumber) Values ("clientAdditionalHeaders","Extensions to the Content-security policy allow only for self-hosted content", 1, 0, "text", 1, "Content-Security Extentions:", 23);
+INSERT INTO Config (ConfigID,Value) VALUES ((SELECT ID FROM ConfigSettings WHERE Name='clientAdditionalHeaders'),"");
```

Comment:
The name isn't very clear. Can you call it something more specific like "CSPAdditionalHeaders"?